tracee-ebpf/go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/aquasecurity/tracee/tracee-ebpf
 go 1.16
 
 require (
-	github.com/aquasecurity/libbpfgo v0.1.2-0.20210803032413-c500b6207bf6
```

Comment:
I suppose this works but doesn't feel right? this commit has nothing to do with v0.1.2 right?

Reply:
This was auto generated by git mod tidy... and it might use the 'next' version as a prefix.. but take in consideration only the commit id at the end (just a guess based on package versioning from distros).